pkg/daemon/update.go

```diff
@@ -1690,6 +1703,41 @@ func (dn *Daemon) updateSSHKeys(newUsers []ign3types.PasswdUser) error {
 	return nil
 }
 
+func deconfigureAbsentUsers(newUsers, oldUsers []ign3types.PasswdUser) {
+	glog.Info("checking to deconfigure the absent user")
+	for _, oldUser := range oldUsers {
+		if !isUserPresent(oldUser, newUsers) {
+			glog.Infof("deconfiguring the user %s\n", oldUser.Name)
+			deconfigureUser(oldUser)
+		}
+	}
+}
+
+func isUserPresent(user ign3types.PasswdUser, userList []ign3types.PasswdUser) bool {
+	glog.Info("checking if user is present")
+	for _, u := range userList {
+		if u.Name == user.Name {
+			return true
+		}
+	}
+	return false
+}
+
+func deconfigureUser(user ign3types.PasswdUser) error {
+	glog.Info("deconfiguring the absent user")
+
+	// clear out password and sshkey
```

Comment:
should this be just password because this function is not making any reset to ssh key?